experimental/Schemes/IdealSheaves.jl

```diff
@@ -891,3 +891,50 @@ function show_details(io::IO, I::IdealSheaf)
     println(io," ")
   end
 end
+
+function _separate_disjoint_components(comp::Vector{<:IdealSheaf}; covering::Covering=default_covering(scheme(first(comp))))
+  isempty(comp) && error("list of components must not be empty")
+  X = scheme(first(comp))
+  all(x->scheme(x) === X, comp) || error("components must be defined over the same scheme")
+  isone(length(comp)) && return covering
```

Comment:
this line is offending

Reply:
What you seem to do is to assure that each patch contains at most one singular point.
This is good. But I want the extra condition that each singular point is visible in EXACTLY one patch. 
So we need to add a second refinement step.

Reply:
May I ask again about the precise task:
You have an IdealSheaf of a 0-dimensional subscheme of some covered scheme X and want to find a covering of X such that each point in the support of your 0-dim. subscheme appears in precisely one chart. Do you want to still cover all of X, possibly allowing charts with no points? Are we still in the surface case or do you need it in more general context?

Reply:
should still cover the scheme. Need it to resolve a surface.
But if the same point appears in two charts, then each blowup is performed twice.
This leads to unnecessarily many charts.